tests/console/hwloc_testsuite.pm

```diff
@@ -0,0 +1,50 @@
+# SUSE's hwloc tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Run testsuite included in hwloc sources
+# Maintainer: Thomas Blume <tblume@suse.com>
+
+use base "consoletest";
+use warnings;
+use strict;
+use testapi;
+use utils;
+
+sub run() {
+    # install hwloc testsuite
+    become_root();
+    if (check_var('VERSION', '12-SP2')) {
+        zypper_call("ar http://download.suse.de/ibs/QA:/SLE12SP2/update/ hwloc-testrepo");
+    }
+    elsif (check_var('VERSION', '12-SP3')) {
+        zypper_call("ar http://download.suse.de/ibs/QA:/SLE12SP3/update/ hwloc-testrepo");
+    }
+
+    zypper_call("ar  http://download.suse.de/ibs/Devel:/HPC:/SLE12SP2/standard/ HPC-module-hwloc");
+    zypper_call("--gpg-auto-import-keys ref");
+    zypper_call("in hwloc hwloc-testsuite");
```

Comment:
Please use single-ticks to be save when you don't want to expand any variables -> `'`